modules/passkey/contracts/SafeWebAuthnSignerProxy.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+import {IP256Verifier} from "./interfaces/IP256Verifier.sol";
+
+/**
+ * @title SafeWebAuthnSignerProxy
+ * @dev A proxy contract that points to SafeWebAuthnSigner.
+ */
+contract SafeWebAuthnSignerProxy {
+    uint256 internal immutable X;
+    uint256 internal immutable Y;
+    IP256Verifier internal immutable VERIFIER;
+    address internal singleton;
```

Comment:
This also needs to be immutable for 4337.

Reply:
https://github.com/safe-global/safe-modules/pull/370/commits/8f613e69f3309c6ae7c0b5772906f9fa1a0735e0